src/user/messages.rs

```diff
@@ -8,18 +8,34 @@ use crate::message::MessageResponder;
 
 #[derive(Deserialize, Serialize)]
 #[serde(tag = "type")]
-pub enum UserMessage {
+pub enum UserQueriesMessage {
     ActiveAuthorsQuery(ActiveAuthorsQuery),
     ActiveReviewersQuery(ActiveReviewersQuery),
 }
 
+#[derive(Deserialize, Serialize)]
+#[serde(tag = "type", content = "payload")]
+pub enum UserMessage {
+    UserActivityByTypeQuery(UserActivityByTypeQuery),
+}
```

Comment:
Why do we need a new namespace for that? So far, all messages in module `Foo` are "exported" together as `FooMesssage`.

Reply:
`ActiveAuthorsQuery` does not have a payload and thus I cannot store it in an enum with `content=payload`. Otherwise we would need to change api.serlo.org so that it sends `payload = {}` for `ActiveAuthorsQuery`.

Reply:
Check. Okay, that makes sense for now.